src/features/firstLogin/tutorials/components/DotComponent.test.tsx

```diff
@@ -1,21 +1,62 @@
-import { render } from '@testing-library/react-native'
+import { fireEvent, render } from '@testing-library/react-native'
 import React from 'react'
 
-import { DotComponent } from 'features/firstLogin/tutorials/components/DotComponent'
+import { DotComponent, getColor } from 'features/firstLogin/tutorials/components/DotComponent'
+import { ColorsEnum } from 'ui/theme'
 
 describe('<DotComponent />', () => {
+  const onPress = jest.fn()
+  beforeEach(jest.clearAllMocks)
+
   it('should render correctly', () => {
     const renderAPI = renderDotComponent()
     expect(renderAPI).toMatchSnapshot()
   })
+
   it('renders dot active color', () => {
     const renderAPI = renderDotComponent({
+      activeIndex: 1,
+      index: 1,
       isActive: true,
+      onPress,
     })
     expect(renderAPI).toMatchSnapshot()
   })
+
+  it('renders trigger onPress when pressed', async () => {
```

Comment:
Ce test a vraiment besoin d'être async ?
```suggestion
  it('renders trigger onPress when pressed', () => {
```